nomad/testing.go

```diff
@@ -67,9 +82,6 @@ func TestServer(t testing.T, cb func(*Config)) (*Server, func()) {
 	f := false
 	config.VaultConfig.Enabled = &f
 
-	// Squelch output when -v isn't specified
```

Comment:
Does this change remove the squelching of output?

Reply:
don't believe so - and if it changed it would have changed due to https://github.com/hashicorp/nomad/pull/8059 not this PR.